go.mod

```diff
@@ -17,8 +18,10 @@ require (
 )
 
 require (
+	github.com/appuio/appuio-cloud-reporting v0.5.0 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/cloudscale-ch/cloudscale-go-sdk/v2 v2.0.1 // indirect
```

Comment:
This shouldn't be necessary as cloudscale-metrics-collector/pkg doesn't actually use the cloudscale-go-sdk. But I'm not sure if you can fix this, maybe this is just go weirdness.

Reply:
The ones with //indirect are managed by `go mod` and indicate a transient dependency. However, issueing `go mod tidy` should cleanup all dependencies.
To figure out why a dependency is needed, you can query it with `go mod why -m github.com/cloudscale-ch/cloudscale-go-sdk/v2`. In this case you'll notice it's not needed, so @ZuGao please issue `go mod tidy` to cleanup unused stuff

Reply:
I had issues with `go mod tidy` and getting dependencies right. Go would require some dependencies then it would clean them for me with `go mod tidy` and the cycle would repet.